front-script/src/containers/main-container/index.js

```diff
@@ -19,8 +19,43 @@ class YakMainContainer extends MetaContainer {
 		global.storage.on('SING-UP-REQ', this.handleSignEvent.bind(this));
 		this.content.append(header, this.input, this.viewer);
 		this.createSignUpForm();
+		this.getParameters();
 		return this.content;
 	}
+	/**
+	 * API get client parameters
+	 */
+	getParameters () {
+		const base = document.querySelector('#yak-chat-embended');
+		this.aplyStyles(base);
+		const that = this;
+		var observer = new MutationObserver(function(mutations) {
+			mutations.forEach(function(mutation) {
+				if (mutation.type == "attributes") {
+					that.aplyStyles(base);
+				}
+			});
+		});
+		observer.observe(base, {
+			attributes: true 
+		});
+	}
+	/**
+	 * aply the styles from the custom attributes
+	 * @param {HTMLElement} base 
+	 */
+	aplyStyles(base) {
```

Comment:
"aply" typo?

Reply:
Yes you're right I meant "apply", I will change it